api/api.go

```diff
@@ -110,7 +111,10 @@ func WithAttachments(data map[interface{}]interface{}) EntryOption {
 // Entry is the basic API of Sentinel.
 func Entry(resource string, opts ...EntryOption) (*base.SentinelEntry, *base.BlockError) {
 	options := entryOptsPool.Get().(*EntryOptions)
-	options.slotChain = globalSlotChain
+	options.slotChain = misc.GetResourceSlotChain(resource)
+	if options.slotChain == nil {
```

Comment:
If options.slotChain is nil, it should be passed directly?
(Mean no rules to check)

Reply:
If failed, using globalSlotChain instead.